index.html

```diff
@@ -1169,19 +1169,66 @@ <h2>Relative DID URLs</h2>
 
   </section>
 
-  <section class="informative">
+  <section>
     <h1>Data Model</h1>
-
+    <p>
+This specification defines an abstract data model for <a>DID documents</a> so it
+is independent of any specific representation. In this section we provide a
+high-level description of the data model, a set of requirements for
+representations, and a set of requirements for extensibility.
+    </p>
     <section>
       <h2>Definition</h2>
+      <p>
+A <a>DID document</a> consists of a collection of name-value pairs which will be
+referred to as properties in this document. The definitions of each of these
+properties are specified in section <a href="#core-properties"></a>. Specific
+representations are defined in section <a href="#core-representations"></a>.
+      </p>
     </section>
-
     <section>
-      <h2>Extensibility</h2>
+      <h2>Representations</h2>
+      <p>
+Following are the requirements for conformant representations.
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding of all
+properties defined in this specification such that the value of any property
+may be losslessly converted into another representation.
+        </li>
+        <li>
+The representation MUST have an unambiguous MIME type.
+        </li>
+        <li>
+The representation MUST define fragment processing rules for its MIME type that
+are conformant with this specification.
+        </li>
+      </ol>
     </section>
-
     <section>
-      <h2>Representation Requirements</h2>
+      <h2>Extensibility</h2>
+      <p>
+The data model supports two types of extensibility.
+      </p>
+      <ol>
+        <li>
+Registry-based extensions are RECOMMENDED for maximum interoperability. The use
+of the registry for properties is the only guaranteed method for two independent
```

Comment:
guaranteed seems a bit strong here. The implementations could still encounter bugs. 

Maybe something along the lines of "The use of the registries makes getting two independent implementations to work together far easier."

Reply:
What about "is the only specified method"
Or "is the only official method"?

Reply:
@kdenhartog I incorporated this language directly from @jricher in #207. @jricher, what do you think?

Reply:
I like where Brent went with this. Of note, I'm not formally objecting to use of the word guaranteed, just thought it felt overly strong.

Reply:
+1 "the only specified method", or perhaps "the most reliable method"

Reply:
+1 to "the only specified method" or even weaker such as "one specified method". Extensions may be interoperable between different representations through mechanisms that don't require the Registry at all.

Reply:
Made this change in commit [77a9787](https://github.com/w3c/did-core/pull/241/commits/77a97872959692d262c16bbd827f857677ba8f82) to fix this wording.

Reply:
Thanks looks good.